ci-deploy/outside-container.sh

```diff
@@ -18,6 +18,8 @@ TRAVIS_PULL_REQUEST=${TRAVIS_PULL_REQUEST:-false}
 
 git clone https://github.com/whatwg/wattsi.git wattsi
 
+git clone https://github.com/pts/pdfsizeopt.git pdfsizeopt
```

Comment:
Maybe we should add a --depth parameter to make these clones take less time.

Reply:
I found https://github.com/whatwg/wattsi/blob/3b651a051a2817bac1b003caf566ce07aa5e2e09/src/build.sh#L21 which made it seem like it shouldn't be added to the first line, but then .git is in .dockerignore, so wattsi --version was already 0 in the existing builds. Adding to both.